indra/newview/llvoavatarself.cpp

```diff
@@ -1350,6 +1364,260 @@ void LLVOAvatarSelf::updateMotions(LLCharacter::e_update_t update_type)
     }
 }
 
```

Comment:
Sending the animation stream is now done by `LLVOAvatarSelf`, not by `LLPuppetMotion`.  All other avatars will only receive animation streams and use `LLStreamingMotion` to play them.